### PR TITLE
update: make improvements to containers

### PR DESCRIPTION
**Pull request by codekow**
- run as user 1001
- remove bind mounts for src (issues with building on various platforms)
- /app as home
- add group write (g+w)
- use fqdn for FROM

### CODE REVIEW

File: Dockerfile.tor-daemon [hunk 1]
```diff
@@ -2,3 +2,12 @@ ARG TOR_VERSION="0.4.7.13"
 ARG TOR_IMAGE="quay.io/bugfest/tor"
 
 FROM ${TOR_IMAGE}:${TOR_VERSION} as tor
+
+USER 0
+
+WORKDIR /app
+RUN chmod g+w .
```
Comment (lexfrei) [inline Dockerfile.tor-daemon:9]: `RUN chmod g+w /app`
Don't use relative path if possible

File: Dockerfile.tor-onionbalance-manager [hunk 2]
```diff
@@ -1,22 +1,30 @@
...
 
-ARG VERSION=0.2.2
+RUN apk add --update --no-cache \
+    git python3 py3-pip py3-wheel py3-cryptography py3-setproctitle py3-pycryptodomex
+RUN python3 -m pip --no-cache-dir install git+https://gitlab.torproject.org/tpo/core/onionbalance.git@${OB_VERSION}
 
-RUN apk add --update git python3 py3-pip py3-wheel py3-cryptography py3-setproctitle py3-pycryptodomex
-RUN python3 -m pip install git+https://gitlab.torproject.org/tpo/core/onionbalance.git@${VERSION}
+WORKDIR /app
+RUN chmod g+w .
```
Comment (lexfrei) [inline Dockerfile.tor-onionbalance-manager:24]: `RUN chmod g+w /app`
Don't use relative path if possible

Comment (codekow) [reply Dockerfile.tor-onionbalance-manager:24]: I'll rebase my branch and make the recommended updates

**Comment by codekow** [comment]
Hopefully the PR isn't too much change. Mainly trying to move toward rootless and a consistent `/app` path. Thanks for the feedback. 👍

File: Dockerfile.tor-onionbalance-manager [hunk 3]
```diff
@@ -23,6 +25,7 @@ RUN apk add --no-cache --update \
         python3=3.10.11-r0 \
     && python3 -m pip install --no-cache-dir git+https://gitlab.torproject.org/tpo/core/onionbalance.git@${VERSION}
```
Comment (lexfrei) [inline Dockerfile.tor-onionbalance-manager:26]: VERSION => OB_VERSION

Comment (bugfest) [reply Dockerfile.tor-onionbalance-manager]: Just committed the fix, let's see the new build

**Pull request merged**